### DISCUSSION TITLE
v0.1.9 - Easy Launcher

### DISCUSSION THREAD

**Discussion by CreativeCodeCat** [DroidWorksStudio/EasyLauncher#94] [Check for Updates]
# What's Changed

### Implemented Enhancements:

- Added support for open app via gesture. - ([e600252](https://github.com/DroidWorksStudio/EasyLauncher/commit/e6002526020a2c19c4dbb094ac830ed458680fbc))

### Bug Fixes:

- Fix. Fixed location permissions - ([42a806d](https://github.com/DroidWorksStudio/EasyLauncher/commit/42a806d15368d802f0c01b152d3adc588577b5e1))
- Fix. Fixed onProviderEnabled and onProviderDisabled for LocationListener - ([60374fe](https://github.com/DroidWorksStudio/EasyLauncher/commit/60374fe6b1634e10c5fdd85bf369b6fdd9fd6476))
- Fixed nightly builds to not overlap with other versions - ([e8d6a9a](https://github.com/DroidWorksStudio/EasyLauncher/commit/e8d6a9abf5463ea5f7670c40962f5a61a3fa2787))
- Fixed the download update to install the internet build. - ([124c4d9](https://github.com/DroidWorksStudio/EasyLauncher/commit/124c4d91aecedabe2c19b58cd09e5e315c407f92))

### Language Support:

- Updated Language Files. ([#91](https://github.com/DroidWorksStudio/EasyLauncher/issues/91)) - ([ceeba28](https://github.com/DroidWorksStudio/EasyLauncher/commit/ceeba281e3b47a170524413b4adfac98d1de36dd))

### Refactoring:

- Added more information onto the README - ([db40f39](https://github.com/DroidWorksStudio/EasyLauncher/commit/db40f39edc2cc0d1dc1abe328b865e21acd1310c))
- Cleanup workflow files - ([43c51a9](https://github.com/DroidWorksStudio/EasyLauncher/commit/43c51a9e35f7c2100ae2f4bf30885820d8434dc5))
- Removed unused code. - ([6034d34](https://github.com/DroidWorksStudio/EasyLauncher/commit/6034d34827704e11665b7ebe4339a5e4327c2b86))

### Releases:

- Prepare for version 0.1.9 - ([4ed1592](https://github.com/DroidWorksStudio/EasyLauncher/commit/4ed159262be81a870b89ab2d134184de895580b2))

**Full Changelog**: https://github.com/DroidWorksStudio/EasyLauncher/compare/0.1.8...0.1.9

<hr /><em>This discussion was created from the release <a href='https://github.com/DroidWorksStudio/EasyLauncher/releases/tag/0.1.9'>v0.1.9 - Easy Launcher</a>.</em>

**Comment by due3die** [comment]
Hi, Hope you are doing well!

I am facing a peculiar problem when I install any app from any source, in the launcher search, it shows duplicate names.



https://photos.app.goo.gl/f2ua8E91dfPt6jWTA

**Comment by due3die** [comment]
> Hi, Hope you are doing well!
> 
> I am facing a peculiar problem when I install any app from any source, in the launcher search, it shows duplicate names.
> 
> 
> 
> https://photos.app.goo.gl/f2ua8E91dfPt6jWTA

https://photos.app.goo.gl/TQKq3AGJkbDB4FGaA

**Comment by due3die** [comment]
What is this email ID droidworksstuido@063240.xyz from app feedback I am unable to reach you, email delivery failed.

**Comment by due3die** [comment]
![Screenshot_2024-07-14-06-41-27-53_38df8cf93173bec2b62fc060ee8450a7.jpg](https://github.com/user-attachments/assets/a08b8fe3-16d7-4b69-af37-af6b61e8caa0)

All app names were showing double double... In the nightly build.

**Reply by CreativeCodeCat** [reply]
Let me guess..... Miui phone?

**Reply by due3die** [reply]
OnePlus 11

**Reply by due3die** [reply]
Is it possible to change the font color in app settings? In Light Mode, the black font doesn't look good against dark wallpapers.

![Screenshot_2024-07-14-19-41-45-80_1d151929efb2ddc7073e99a7f966a46b.jpg](https://github.com/user-attachments/assets/8db8a21a-f3c8-4cff-bd18-c0aa7306a288)

![Screenshot_2024-07-14-19-43-55-26_1d151929efb2ddc7073e99a7f966a46b.jpg](https://github.com/user-attachments/assets/2af18e09-2e76-4f61-881f-72e1efb5760d)

![Screenshot_2024-07-14-19-42-32-55_1d151929efb2ddc7073e99a7f966a46b.jpg](https://github.com/user-attachments/assets/93963b2a-1054-44b5-a8bc-1ef7744867af)

![Screenshot_2024-07-14-19-43-50-81_1d151929efb2ddc7073e99a7f966a46b.jpg](https://github.com/user-attachments/assets/e4739345-223e-46bf-b282-a7421c6e10a9)

**Reply by CreativeCodeCat** [reply]
no i am not going to add settings text color as that's only seen when changing settings not seen all the time.

**Reply by due3die** [reply]
So how do we see that?

Can we give a solid background so it will not mess with wallpapers?

**Reply by due3die** [reply]
> no i am not going to add settings text color as that's only seen when changing settings not seen all the time.

![IMG_20240715_060917.jpg](https://github.com/user-attachments/assets/149d1243-20af-47fa-a3bb-352d94ede789)

See the attached file, some font color messing with the background other than settings.

![Screenshot_20240715_061553.png](https://github.com/user-attachments/assets/0bce4a03-fd62-45ac-a389-fd76c49758e2)

**Reply by CreativeCodeCat** [reply]
Ya the color issue will be fixed in 0.2.0

**Reply by CreativeCodeCat** [reply]
> So how do we see that?
> 
> Can we give a solid background so it will not mess with wallpapers?

I'll change the opacity for settings so that it's lighter or darker :) so you only see 10% opacity of the background ATM it's at 90%
